plugins/board-resources/src/utils/CardUtils.ts

```diff
@@ -67,3 +67,20 @@ export function archiveCard (card: Card, client: Client): Promise<TxResult> | un
 export function unarchiveCard (card: Card, client: Client): Promise<TxResult> | undefined {
   return updateCard(client, card, 'isArchived', false)
 }
+
+export function updateCardMembers (
```

Comment:
Better to use one transaction with both $push: { members: [newMembers] } and $pull: { members: [oldMemebrs]}. Please check if it working, or at least two transactions instead of multiple ones. 
Also pleas make function async and return Promise<void>, it will be faster and less transactions.

Reply:
`$pull` doesn't support array
https://github.com/hcengineering/anticrm/blob/eb9526f0153f10550c1eb537dd8c176d31921cf8/packages/core/src/operator.ts#L42
Also using both operators cause
`ERROR: platform:status:UnknownError {"message":"Updating the path 'members' would create a conflict at 'members'"}`

Reply:
I suppose $pull: {values: {$in: array}} should work, at least for mongoDb, 
if so we could fix it for our $pull implementation.

Reply:
Done